ci/jenkins/jobs/macros.yaml

```diff
@@ -223,20 +223,23 @@
           set -e
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test-vm.sh  --registry ${DOCKER_REGISTRY} --kubeconfig /var/lib/jenkins/.kube/config
-          
+
 - builder:
-    name: builder-conformance-kind
```

Comment:
It seems this patch changes existing builder builder-conformance-kind. It should create a new builder.